addon/pods/month-picker/template.hbs

```diff
@@ -1,15 +1,20 @@
 <div class="dp-month-picker">
-  {{#if showInput}}
-    {{range-input-mask value=rangeFormatted
-                       mask="9[9]/99[99]—9[9]/99[99]"
-                       class="dp-date-input"
-                       focus-in="toggleIsExpanded"
-                       key-press="parseInput"
-                       tabindex=1}}
-  {{/if}}
-
-  {{#if isExpanded}}
-    <div class="dp-panel">
+  {{#basic-dropdown verticalPosition="below"
+                    registerAPI=(action (mut dropdownController))
+                    initiallyOpened=initiallyOpened
+                    renderInPlace=true
+                    as |dropdown|}}
+    {{#dropdown.trigger tabindex=tabIndex
+                        onFocus=(action "onFocusInput")
+                        onKeydown=(action "handleKeydown")}}
+      {{range-input-mask value=rangeFormatted
+                         mask="9[9]/99[99]—9[9]/99[99]"
+                         class="dp-date-input"
+                         key-press="parseInput"}}
+    {{/dropdown.trigger}}
+  
+    {{#dropdown.content class="dp-panel"
+                        to="dp-month-picker"}}
       <div class="dp-display-month-year">
```

Comment:
`classNames` could probably also be applied here as well as the other types of pickers